code/modules/mob/living/simple_animal/borer/borer_powers.dm

```diff
@@ -100,10 +105,18 @@
 	if(!M.stat)
 		to_chat(M, SPAN_DANGER("Something disgusting and slimy wiggles into your ear!"))
 
+	if(invisibility)
+		src.invisibility = 0
+		src.alpha = 255
+
+	if(sight & SEE_MOBS)
+		sight &= ~SEE_MOBS
+		to_chat(src, SPAN_NOTICE("You cannot see living being throug walls for now."))
```

Comment:
`to_chat(src, SPAN_NOTICE("You cannot see living being throug walls for now.")) `

Should be "through walls" instead of "throug walls" I believe